logrotate/juju-db.conf

```diff
@@ -0,0 +1,12 @@
+/var/snap/juju-db/common/logs/mongodb.log {
+    weekly
+    rotate 5
+    compress
+    delaycompress
+    missingok
+    notifempty
+    create 0644 root root
+    postrotate
+        snapctl restart juju-db.daemon
```

Comment:
Would this cause outages / downtime to the mongodb?

Instead, perhaps using copytruncate?

Reply:
Yes, downtime is possible. I updated the configuration with the `copytruncate` option.